editor.planx.uk/src/pages/FlowEditor/components/Subscription/components/Contract.tsx

```diff
@@ -0,0 +1,16 @@
+import Typography from "@mui/material/Typography";
+import React from "react";
+import SettingsSection from "ui/editor/SettingsSection";
+
+export const Contract = () => (
+  <SettingsSection background>
+    <Typography variant="h3" component="h4" gutterBottom>
+      Contract
+    </Typography>
+    <Typography variant="body2" color="text.secondary">
+      Terms and key dates of your software contract. OSL invoices annually for
+      the fixed subscription cost of Plan✕.
+    </Typography>
+    <Typography variant="body1">[Coming soon]</Typography>
```

Comment:
Intentionally "Coming soon" to illustrate direction of travel - but we ultimately need to revisit this one in a follow-up PR and decide how to integration Notion CRM data points like amount, start and end date !